apps/patient/src/components/PharmacyFilters.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, HStack, Text } from '@chakra-ui/react';
+
+interface PharmacyFiltersProps {
+  enableOpenNow: boolean;
+  enable24Hr: boolean;
+  setEnableOpenNow: (isOpenNow: boolean) => void;
+  setEnable24Hr: (is24Hr: boolean) => void;
+}
+export const PharmacyFilters = ({
+  enableOpenNow,
+  enable24Hr,
+  setEnableOpenNow,
+  setEnable24Hr
+}: PharmacyFiltersProps) => {
+  return (
+    <HStack>
+      <Text>Filter by</Text>
+      <Button
+        size="sm"
+        variant="filter"
+        isActive={enableOpenNow}
+        onClick={() => {
+          if (enable24Hr) setEnable24Hr(false);
```

Comment:
Filters are mutually exclusive since openNow and open24Hr is kind of weird

Reply:
I think it's fine to allow clicking both and I think it'd be a bit weird if clicking "openNow" turned off `24hr` filter

Reply:
Either way, you can do `setEnable24Hr(false)` without having the `if` statement